core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java

```diff
@@ -59,15 +58,7 @@ public DataPoint(
         tags = Collections.emptyMap();
     }
 
-    @JsonCreator(mode = Mode.PROPERTIES)
-    public DataPoint(
```

Comment:
I do not remember now but there was a reason for having just the constructor annotated. I really need to remember this ....